apps/web/components/CardWithFeaturedItems/CardWithFeaturedItems.css.ts

```diff
@@ -0,0 +1,24 @@
+import { style } from '@vanilla-extract/css'
+import { themeUtils } from '@island.is/island-ui/theme'
+
+export const container = style({
+  height: 260,
+  minWidth: 0,
+  minHeight: 0,
+})
```

Comment:
I wonder if adding overflow: 'hidden' here might be better since at some screen sizes the images go outside the card
![image](https://github.com/island-is/island.is/assets/43557895/d31c9577-dcd3-41df-90a2-d35fe5644a54)